smart-contracts/contracts/BridgeBank/BridgeBank.sol

```diff
@@ -289,6 +290,23 @@ contract BridgeBank is BankStorage,
         unlockFunds(_recipient, tokenAddress, symbol, _amount);
     }
 
+    event LogMigrateRowan(
+        address migrator,
```

Comment:
Please index the fields you think should be important to index.

I think `migrator` and `amount` at the very least should be indexed in this event.

Reply:
We can only have 3 indexed arguments at most and I wonder how we're going to use `amount` in our systems.
In principle, I would rather index `migrator`, `previousToken`, and `newToken`, because each user will be able to do only one migration (right?). We could query the events using `migrator` as key, instead of `amount`.